### DISCUSSION TITLE
When will the TarOutputStream asynchronous APIs be released?

### DISCUSSION THREAD

**Discussion by lingbohome** [icsharpcode/SharpZipLib#764] [Q&A]
I am on the Master branch and see that TarOutputStream already supports asynchronous apis. When do you plan to release the new Nuget package?

**Comment by piksel** [comment]
I am currently sick, but I plan on creating a release as soon as the fever goes down.

**Reply by lingbohome** [reply]
Ok, thank you.
I wish you a speedy recovery.
